scripts/profiler/plot_nordic.py

```diff
@@ -526,7 +525,7 @@ def plot_events_real_time(
         self.start_stop_button.on_clicked(self.on_click_start_stop)
         plt.sca(self.draw_state.ax)
 
-        ani = animation.FuncAnimation(
+        self.ani = animation.FuncAnimation(
```

Comment:
This isn't a linter change, is it?

Reply:
According to matplotlib documentation:
We need to keep the reference to an `FuncAnimation` object here. Otherwise the animation may not work properly - the object may be garbage collected.

Reply:
I'm not arguing with the need for this change, just saying that it's not a linter change.

Can you please separate this into a bugfix commit?

Reply:
Ok